sched/Kconfig

```diff
@@ -1808,14 +1808,13 @@ menu "Stack and heap information"
 
 config DEFAULT_TASK_STACKSIZE
 	int "The default stack size for tasks"
-	default 65536 if ARCH_SIM
 	default 2048
 	---help---
 		The default stack size for tasks.
 
 config IDLETHREAD_STACKSIZE
 	int "Idle thread stack size"
-	default DEFAULT_TASK_STACKSIZE if ARCH_SIM
+	default 65536 if ARCH_SIM
 	default 1024
```

Comment:
let's change default to DEFAULT_TASK_STACKSIZE?

Reply:
we can remove this "default DEFAULT_TASK_STACKSIZE if ARCH_SIM" by up_initial_state adjust.